src/robusta/core/playbooks/prometheus_enrichment_utils.py

```diff
@@ -220,6 +231,14 @@ def create_resource_enrichment(
         if title_override
         else f"{resource_type.name} {values_format_text} for this {item_type.name.lower()}"
     )
+
+    # NOTE: Some queries do not produce automatic labels, so we need to provide them
+    # Parameter in lambda is the number of the series in the chart to override (excluding lines)
+    # It could be used if there are multiple series in the chart
+    chart_label_factories: Dict[ResourceKey, ChartLabelFactory] = {
+        (ResourceChartResourceType.Memory, ResourceChartItemType.Container): lambda i: "Memory Usage",
```

Comment:
Is it right to add the `container` label here?
For this query you have the container label in the query ($container)
I think the chart title is "Memory Usage"
WDYT?